net/gatling/patches/010-decrease_optimisation.patch

```diff
@@ -0,0 +1,25 @@
+gatling: decrease optimisation
+
+When compiling on MIPS-architectures, in -O2 there happen some
```

Comment:
This can be disabled only for MIPS, right? But I still dont think that this is a good solution at all as you referenced two years old issue and it seems only one person has this. Maybe let's cc: @neheb on this one.

Reply:
As far as I saw it on my compile test, MIPS would be the only target that needs this patch. I don't know how to apply a patch for specific target only. Would be great, if you could provide me with some instructions on that.

For the one issue 2 years ago: I could have linked some 3 website more. If the users did solve the issue at all, they mostly decreased optimization level. This site served as one example. You can find all of these by googling the linkers error.
https://www.google.com/search?q=gcc+Unsupported+jump+between+ISA+modes%3B+consider+recompiling+with+interlinking+enabled

If it helps, there's an overview on compiler-flags, that get activated by `-O2`:
https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html